avocado/utils/wait.py

```diff
@@ -4,7 +4,7 @@
 log = logging.getLogger('avocado.test')
 
 
-def wait_for(func, timeout, first=0.0, step=1.0, text=None):
+def wait_for(func, timeout, first=0.0, step=1.0, text=None, args=[], kwargs={}):
```

Comment:
Please remember not using mutable types as default arguments (modifications to them carry to next usages).

I'm taking care of fixing this one and adding it to our inspect check (as it looks like we have quite many of them)

Reply:
https://github.com/avocado-framework/avocado/pull/2323

Reply:
Thanks for the fix. You are right.

Reply:
no problem, it was not the only place and now we have an inspect check 😉